### DISCUSSION TITLE
示列 还是加载不出来

### DISCUSSION THREAD

**Discussion by Xingen123** [zouyaoji/vue-cesium#277] [Q&A]
示列 还是加载不出来

**Comment by zouyaoji** [comment]
http://zouyaoji.top/vue-cesium-demo

是说的这个吗？
已经可以了哈。

**Reply by Xingen123** [reply]
https://vue-cesium.songluck.com/#/zh-CN/component/vc-viewer  这个

**Reply by zouyaoji** [reply]
看来是自动同步出了问题，我手动同步一下。

**Reply by zouyaoji** [reply]
是gitee的page需要实名认证了，晚些认证一下，你先看 github的吧。

https://zouyaoji.top/vue-cesium/#/zh-CN/component/vc-viewer

**Reply by zouyaoji** [reply]
好了哈。
